sgv2-service-common/src/main/java/io/stargate/sgv2/common/cql/builder/Predicate.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright DataStax, Inc. and/or The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.common.cql.builder;
+
+public enum Predicate {
```

Comment:
While non-ambiguous within context, I wonder if it'd be good to name it `CqlPredicate` to distinguish from among other types. Similar questions about `Value`. I guess it depends on whether this would be exposed beyond immediate building of CQL query/statement or not -- if not, does not really matter. But if type is used and passed could sometimes be confusing.